tapir/wirgarten/tapirmail.py

```diff
@@ -85,58 +87,24 @@ def _register_segments():
     base = Member.objects.all()
     register_base_segment(base)
 
-    def get_member_ids(annotation, filter_condition):
-        return base.filter(
-            id__in=Member.objects.annotate(**annotation)
-            .filter(**filter_condition)
-            .values_list("id", flat=True)
-        )
-
-    shares_annotation = {
-        "total_shares": models.Sum(
-            models.Case(
-                models.When(
-                    coopsharetransaction__valid_at__lte=get_today(),
-                    then=models.F("coopsharetransaction__quantity"),
-                ),
-                default=0,
-                output_field=models.IntegerField(),
-            )
-        )
-    }
-
-    subscription_annotation = {
-        "active_subscription_count": models.Count(
-            models.Case(
-                models.When(
-                    subscription__start_date__lte=get_today(),
-                    subscription__end_date__gte=get_today(),
-                    then=1,
-                ),
-                output_field=models.IntegerField(),
-            )
-        )
-    }
```

Comment:
The annotation definitions that I mention [here](https://github.com/FoodCoopX/wirgarten-tapir/pull/472#issuecomment-2214474072)